pkgs/development/tools/yarn2nix-moretea/yarn2nix/lib/generateNix.js

```diff
@@ -34,18 +34,24 @@ const { execFileSync } = require('child_process')
 
 function prefetchgit(url, rev) {
   return JSON.parse(
-    execFileSync("nix-prefetch-git", ["--rev", rev, url, "--fetch-submodules"], {
-      stdio: [ "ignore", "pipe", "ignore" ],
-      timeout: 60000,
-    })
-  ).sha256
+    execFileSync(
+      "nix-prefetch-git",
+      ["--rev", rev, url, "--fetch-submodules"],
+      {
+        stdio: ["ignore", "pipe", "ignore"],
+        timeout: 60000
+      }
+    )
+  ).sha256;
 }
 
 function fetchgit(fileName, url, rev, branch, builtinFetchGit) {
   return `    {
     name = "${fileName}";
     path =
-      let${builtinFetchGit ? `
+      let${
+        builtinFetchGit
+          ? `
         repo = builtins.fetchGit ({
```

Comment:
Maybe we should extract this into a variable rather than nest the format-strings so that the indentation does not jump weirdly like this.